data/184/752/401/5/1847524015.geojson

```diff
@@ -0,0 +1,55 @@
+{
+  "id": 1847524015,
+  "type": "Feature",
+  "properties": {
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"uuuu",
+    "geom:area":0.007844,
+    "geom:bbox":"9.956353,53.556387,10.0852,53.68199",
+    "geom:latitude":53.617616,
+    "geom:longitude":10.016526,
+    "iso:country":"DE",
+    "lbl:latitude":53.610715,
+    "lbl:longitude":10.01212,
+    "mps:latitude":53.610715,
+    "mps:longitude":10.01212,
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "name:deu_x_preferred":[
```

Comment:
For DEU and ENG `Nord` should be preferred name, and `Hamburg-Nord` a variant (and possibly a long name for disambiguation)